modules/custom-post-types/testimonial.php

```diff
@@ -368,6 +368,9 @@ function updated_messages( $messages ) {
 	 * Change ‘Enter Title Here’ text for the Testimonial.
 	 */
 	function change_default_title( $title ) {
+		if ( ! function_exists( 'get_current_screen' ) ) {
```

Comment:
All this function could be replaced with
```php
if ( self::CUSTOM_POST_TYPE == get_post_type() ) {
    $title = esc_html__( "Enter the customer's name here", 'jetpack' );
}
return $title;
```
I tested this in wp-admin only (01:21 am) but I'm confident it will work in front end as well because `get_post_type` is defined in `post.php` which is always loaded (in `wp-settings.php`, unless it's a SHORTINIT which is not the case here) so we would avoid checking that the function exists, getting the screen object, and finally using the property of the screen object. Since we're only interested in knowing that the current post type is `testimonial`, we can use this function.